### PR TITLE
Removing the package-lock.json files.

### PR DESCRIPTION
**Pull request by agarwalakarsh**
**Ref:**  #90 

**Description:**

`package-lock.json` makes sure that the packages (`npm packages`) when being installed via the `npm install` command is so in a specific sequence of package version. This makes sure that the remaining packages, if dependent on other packages, are not installed `twice` or break due to the unavailability of the packages.

While this is useful in most cases, in our `explorer` it looks like it is not an absolute requirement. By deleting these files, we can reduce the `repo` size by `100s of MBs`. Which in turn will help us in creating the containers locally with speed.

Also, due to its huge size, `cloning` the repo becomes a problem.

**Blocking:** Nah

**Testing:**

For the `blockchain-explorer/client` directory:

- Delete the `node_modules` directory.
- Run the `npm install`.
- Build the `react` code with `REACT_APP_CLIENT=Provider npm run build`.

Now, move to the root directory of the repo, `blockchain-explorer`:

- Delete the `node_modules` directory.
- Run the `npm install`.

Make sure that the `bcmanager` network is up and running. Once that is done, use
`REACT_APP_CLIENT=Provider npm start`
command from the `blockchain-explorer` directory and access the `explorer` on `localhost:11000` address. 

Everything should be working fine.

### CODE REVIEW

**Comment by sallykeating** [comment]
Also I think with this PR, these two file names should be added to .gitignore, because when the explorer is run the files (`package-lock` and `client/package-lock`) are re-created, meaning that each time you try to navigate to another branch you're stopped suggesting to stash these files. So if we add them to .gitignore it will save us from having to stash them all the time

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,5 @@ fabric-docker-compose-svt/crypto-config
 first-network/channel-artifacts/*
 .vscode
 log.txt
+package-lock.json
```
Comment (sallykeating) [inline .gitignore]: Awesome!! and this will do both the `client/package-lock.json` and the main one?

Comment (agarwalakarsh) [reply .gitignore]: I have added inside the `client` directory too now. That should do it finally. 🍡

**Comment by agarwalakarsh** [comment]
Yayy!! 🔥

**Pull request merged**